app/presenters/tree_builder_belongs_to_hac.rb

```diff
@@ -50,7 +50,6 @@ def set_locals_for_render
     locals.merge!(:oncheck           => oncheck,
                   :check_url         => check_url,
                   :highlight_changes => @assign_to ? false : true,
-                  :checkboxes        => true,
```

Comment:
Why no checkboxes true in `tree_init_options`?

What's the difference between `checkboxes` in options and `checkboxes` in locals, in master?

Reply:
Hm, you're right, here the `true` is simply enough.

Reply:
Oh, I know now, the `checkboxes` in options are being used for setting the checkability of the nodes, but that can be moved into the `override`...fixing